docs/altos.md

```diff
@@ -18,9 +18,9 @@ Pros:
 
 Cons: 
 * Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices
-* Thunderbolt driver is paid (20 USD) for certain generations
+* Thunderbolt driver is paid (20 USD) for devices that **have** them
```

Comment:
I don't like how this is worded, maybe something like:

Certain devices with Thunderbolt ports will need to purchase an additional driver ($20 USD)

Reply:
yeah that sounds better, ill add that in